termwiz/src/render/terminfo.rs

```diff
@@ -807,27 +793,27 @@ mod test {
     }
 
     impl Terminal for FakeTerm {
-        fn set_raw_mode(&mut self) -> anyhow::Result<()> {
-            bail!("not implemented");
+        fn set_raw_mode(&mut self) -> StdResult<(), TerminalError> {
+            unimplemented!()
```

Comment:
this change will make the program panic with no chance to handle the error, whereas previously the caller could decide to handle this

Reply:
yeah, but this is in test code?

Reply:
Undone